README.md

```diff
@@ -32,14 +32,42 @@ pub struct AppInfo {
 ```
 
 ### Setting Up the Bot
+
 ### Requirements
-You need to have `rust` and `node` installed.
 
+You need to have `rust` and `node` installed.
+Optionally, have [pnpm](https://pnpm.io/) installed.
 
 ### Setup
-1. Go to the `frontend` folder and install the packages using `pnpm i`.
-2. Build the frontend using `pnpm build`.
-3. Start the bot (from the root folder) `addr="<email>" mail_pw="<password>" cargo r`
+
+Go to the `frontend` folder and install the packages using `pnpm install` or `pnpm i`.
+If you do not have `pnpm` installed, run `npx pnpm install` instead.
+This step creates a `node_modules` directory.
+
+Build the frontend using `pnpm run build` or `npx pnpm run build`.
+This step creates a `bot-data` directory in the root of the repository
+with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
+
+Set environment variables `addr` and `mail_pw` with the login and password
+of the email account that the bot will use:
+```
```

Comment:
I would personally give onliner which you can just paste in `addr=<> mail_pw=<> cargo r` instead of this "more advanced" usage.

Reply:
But only for first configuration, no?